packages/xdl/src/detach/IosNSBundle.js

```diff
@@ -366,6 +366,13 @@ async function _configureInfoPlistAsync(context: StandaloneContext) {
       infoPlist.UIDeviceFamily = [2];
     }
 
+    // Whether requires full screen on iPad
+    infoPlist.UIRequiresFullScreen = config.ios && config.ios.requireFullScreen;
```

Comment:
If this is not a boolean value it may cause issues -- I think we may want to cast to make sure that it is a boolean.